snapcraft/internal/remote_build/_launchpad.py

```diff
@@ -132,6 +134,8 @@ def start_build(self, timeout: int = 5, attempts: int = 5) -> int:
             )
             if builds.entries:
                 break
+
+            # TODO: there may actually be a real build error here!
```

Comment:
this change seems unnecessary considering #2642

Reply:
Indeed, removed.